.idea/dictionaries/Oleg_Ivanov.xml

```diff
@@ -0,0 +1,7 @@
+<component name="ProjectDictionaryState">
+  <dictionary name="Oleg.Ivanov">
+    <words>
+      <w>dont</w>
```

Comment:
why is this here?

Reply:
Why not? There is already some dictionaries!

Reply:
because "dont" is not grammatically correct, you use don't! you don't use dont!